.github/workflows/ci.yml

```diff
@@ -30,8 +30,8 @@ jobs:
 
     env:
       CI_IMAGE_PREFIX: registry.gitlab.com/buildstream/buildstream-docker-images/testsuite
-      CI_IMAGE_SUFFIX: master-154893333
-      TOXENV: py36,py6-plugins,py37,py37-plugins,py38,py38-plugins
+      CI_IMAGE_SUFFIX: master-177137613
+      TOXENV: py36,py37,py38-nocover,py36-plugins,py37-plugins,py38-plugins-nocover
```

Comment:
It's really annoying that Coverage *still* doesn't play properly with cython in 3.8. There's still the open issue to track this so hopefully it can be re-added eventually https://gitlab.com/BuildStream/buildstream/-/issues/1173 (with 3.9 being released we may well have the same issue)

Reply:
Of course.